### ISSUE TITLE
Set headers based on mimetype

### ISSUE THREAD

**Issue by andydavies** [h2o/h2o#1150]
I'd like to be able to set headers based on the mimetype of the response e.g. only set CSP header on html responses

Just checking I haven't missed something as based on my reading of the docs this isn't currently possible?

**Comment by kazuho** [comment]
You are correct that there's no way of doing such thing at the moment. There has been a discussion to support output filtering using mruby, and for the record I've opened #1159.
